storage.bs

```diff
@@ -331,8 +341,13 @@ must run these steps:
 
    <li><p>Let <var>quota</var> be <a>site storage quota</a> for <var>origin</var>.
 
+   <li><p>Let <var>usageDetails</var> be a new {{StorageUsageDetails}} dictionary
```

Comment:
The _quota_ is connected to this site storage unit. Should that concept be mentioned here, or is this type of usageDetails not related to the site storage unit?